tests/playground.html

```diff
@@ -660,6 +660,7 @@ <h1>Blockly Playground</h1>
     </category>
     <category name="Text" categorystyle="text_category">
       <block type="text"></block>
+      <block type="text_multiline"></block>
```

Comment:
Please also add this to the typed variables toolbox, and the multi-playground categories toolbox.